code/game/machinery/doors/airlock.dm

```diff
@@ -43,6 +43,7 @@ var/list/airlock_overlays = list()
 	name = "airlock"
 	icon = 'icons/obj/doors/airlocks/station/public.dmi'
 	icon_state = "closed"
+	move_resist = MOVE_FORCE_VERY_STRONG
```

Comment:
I'm not too sure really, I just did what TG did. I'm assuming it's to make it so airlocks can't be pushed? Though it would make more sense to be anchored, so I can put it back to that. This was before I understood what move_force and such did, one of the first parts of it I edited :p.

Reply:
Changed it back to anchored.